secure-api-gateway-ob-uk-rs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rs/server/configuration/BigDecimalDefaults.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2020-2022 ForgeRock AS (obst@forgerock.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.forgerock.sapi.gateway.ob.uk.rs.server.configuration;
+
+import java.math.RoundingMode;
+
+/**
+ * Default features for BigDecimals
+ */
+public interface BigDecimalDefaults {
+    int DEFAULT_SCALE = 4;
+    RoundingMode DEFAULT_ROUNDING_MODE = RoundingMode.HALF_EVEN;
+}
```

Comment:
Make this configurable using Spring properties and pass it as a dependency into the serializer/deserializer

Reply:
This has been deleted, we not need anymore the deserializer/serializer or BigDecimalDefaults, now has been implemented a equality util and all generated OB objects with BigDecimal fields use the new util to verify the equality of BigDecimals.